extensions/pyRevitTools.extension/pyRevit.tab/Project.panel/Preflight Checks.pushbutton/config.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: UTF-8 -*-
+
+from os.path import dirname, abspath, join
+from pyrevit import script
+from pyrevit.forms import alert, pick_file, pick_folder
+from csv import reader
+
+script.get_output().close_others()
+
+config = script.get_config()
+
+current_folder = dirname(abspath(__file__))
+config.set_option(op_name='current_folder', value=current_folder)
+
+critical_warnings_filepath = pick_file(init_dir=current_folder,
+                                             file_ext="csv",
+                                             multi_file=False, 
+                                             title="Select the CSV file containing the critical warnings GUIDs")
+if critical_warnings_filepath is None:
+    alert("No CSV file selected. Exiting...", exitscript=True)

```

Comment:
Do we need to create an alert if the user consciously selected "cancel" in the pick file dialog?
```suggestion
    script.exit()
```